capa/features/insn.py

```diff
@@ -13,11 +13,6 @@
 
 class API(Feature):
     def __init__(self, name: str, description=None):
-        # Downcase library name if given
-        if "." in name:
-            modname, _, impname = name.rpartition(".")
-            name = modname.lower() + "." + impname
-
```

Comment:
moved to https://github.com/mandiant/capa/blob/656776f26ef04d3c413a6a62e55d3cd0e85c7cdc/capa/features/extractors/helpers.py#L45